data/emulator.js

```diff
@@ -452,7 +452,10 @@ class EmulatorJS {
         this.elements.parent.appendChild(this.textElem);
     }
     localization(text, log) {
-        if (!isNaN(text)) return text;
+        if (!text) return text;
+        text = text.toString();
```

Comment:
Why are we `toString`ing this?

Reply:
To convert it to a string if it is a number

Reply:
Then this should be before `if (!text)` since `if (!0)` will return true

Reply:
I don't actually think it is necessary

Reply:
Keep the check just in case

Reply:
Ok so it is good as is.

Reply:
(do `text = text.toString` first, and make sure `typeof text !== undefined`